draft-ietf-ppm-dap.md

```diff
@@ -191,81 +191,79 @@ seen in the clear by any server.
 
 {::boilerplate bcp14-tagged}
 
-The following terms are used:
-
 Aggregate result:
-: The output of the aggregation function over a given set of reports.
+: The output of the aggregation function over a given set of measurements and
+  aggregation parameter. As defined in {{!VDAF=I-D.draft-irtf-cfrg-vdaf-03}}.
 
 Aggregate share:
-: A share of the aggregate result emitted by an aggregator. Aggregate shares are
-  reassembled by the collector into the final output.
+: A share of the aggregate result emitted by an Aggregator. Aggregate shares are
+  reassembled by the Collector into the aggregate result, which is the final
+  output of the aggregation function. As defined in {{!VDAF}}.
 
 Aggregation function:
-: The function computed over the users' inputs.
+: The function computed over the Clients' measurements. As defined in {{!VDAF}}.
+
+Aggregation parameter:
+: Parameter used to prepare a set of measurements for aggregation (e.g., the
+  candidate prefixes for Poplar1 from {{Section 8 of !VDAF}}). As defined in
+  {{!VDAF}}.
 
 Aggregator:
-: An endpoint that runs the input-validation protocol and accumulates input
-  shares.
+: An endpoint that recieves input shares from Clients and validates and
+  aggregates them with the help of the other Aggregators. As defined in
+  {{!VDAF}}.
```

Comment:
There's more to a DAP aggregator than VDAF's definition of an aggregator, so perhaps rephrase to explain that a DAP aggregator is also a VDAF aggregator.